docs/usage/static-files.rst

```diff
@@ -1,44 +1,47 @@
 Static files
 ============
 
-To serve static files (i.e. serve arbitrary files from a given directory), the
+To serve static files (e.g., serve arbitrary files from a given directory), the
```

Comment:
I think `i.e.` is correct here. Serving arbitrary files isn't an example of using static files; It's the concept, just a bit more wordy.